internal/resolver/resolver_test.go

```diff
@@ -156,20 +156,36 @@ func TestResolve(t *testing.T) {
 }
 
 func TestHashReq(t *testing.T) {
-	expect := "sha256:d661820b01ed7bcf26eed8f01cf16380e0a76326ba33058d3150f919d9b15bc0"
+	expect := "sha256:96045f8dfd8769a3a1af6202449fa620c900992277803e799fe31c3897f179e8"
 	req := []*chart.Dependency{
-		{Name: "alpine", Version: "0.1.0", Repository: "http://localhost:8879/charts"},
+		{Name: "alpine", Version: "^0.1.0", Repository: "http://localhost:8879/charts"},
```

Comment:
Can we please keep this as-is? I'd rather we keep the existing tests in place and add new tests to the suite rather than replacing existing test cases.